orderer/consensus/smartbft/util_network_test.go

```diff
@@ -552,7 +554,7 @@ func createBFTChainUsingMocks(t *testing.T, node *Node, configInfo *ConfigInfo)
 				return
 			}
 			t.Logf("Node %d requested SendTransaction to node %d", node.NodeId, targetNodeId)
-			err := node.sendRequest(node.NodeId, targetNodeId, message)
+			err = node.sendRequest(node.NodeId, targetNodeId, message)
```

Comment:
why?

Reply:
It seems to me that using existing variables is better than declaring new ones and allocating memory for them.